images/win/scripts/Installers/Install-VS.ps1

```diff
@@ -42,6 +42,12 @@ if (Test-IsWin19) {
 	$sdkFileName = "sdksetup14393.exe"
 	$argumentList = ("/q", "/norestart", "/ceip off", "/features OptionId.WindowsSoftwareDevelopmentKit")
 	Install-Binary -Url $sdkUrl -Name $sdkFileName -ArgumentList $argumentList
+
+	# Install Windows 11 SDK version 10.0.22621.0
```

Comment:
Maybe we should bump SDK version in the Install-WDK.ps1 script, but we should check first.

Reply:
I thought about it, but I'm not sure. The question is what set of options we need in the new SDK. And what happens if we change the main version of SDK. Need to test.